tests/TranslatableTest.php

```diff
@@ -728,36 +727,32 @@ public function it_uses_value_when_fallback_is_not_available()
         $this->assertEquals('Erbsen', $vegetable->getAttribute('name'));
     }
 
-    public function test_translation_with_multiconnection()
+    /** @test */
+    public function translation_model_is_saved_in_the_correct_connection()
     {
-        $this->migrate('mysql2');
-
-        // Add country & translation in second db
-        $country = new Country();
-        $country->setConnection('mysql2');
-        $country->code = 'sg';
-        $country->{'name:sg'} = 'Singapore';
-        $this->assertTrue($country->save());
+        $potatoes = new Vegetable();
+        $potatoes->{'name:en'} = 'Potatoes';
+        $potatoes->save();
 
-        $countryId = $country->id;
+        $this->app->make('config')->set('database.connections.mysql2', [
```

Comment:
I've two ideas here.
1) use test annotation to execute it only on TravisCI
2) check if PHP can handle two memory sqlite
https://www.tekz.io/blog/laravel-phpunit-testing-multiple-connection-system-with-sqlite-in-memory-db

Because it's a super special test I would be fine with executing it only on Travis. I will add the annotation and required method later.

Reply:
The problem is the MySQL - in opposit to sqlite it needs a program/instance/Server to run it. So this test won't work out of the box if someone checks this repo out and want to run phpunit.
In the past you had to use the CircleCI local CLI for tests. Now we are based on sqlite by default and this is great! Because now the only thing you need is PHP (with some extensions). I would like to stick with it and keep the testsuite runnable on a local machine without MySQL.

We do something similar in https://github.com/spatie/laravel-activitylog/blob/091efc48be732e8704e2bd2bc9aa332f78800bb9/tests/TestCase.php#L27-L34

So how about an `@mysql` annotation which checks the current database driver (default connection) and marks this test as skipped if it isn't mysql? The other option would be to setup the second connection depending on the first one.

And because all of this needs "a lot" of logic/code - how about moving it into it's own test class `MultiConnectionTranslatableTest`? This way we can use the `setUp`, `tearDown` hooks and add additional logic without bloating the other tests.

Reply:
100% agree with that. I'll try to configure the second sqlite connection first 👍